scrape.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/pin/python
+from requests import get
+from requests.exceptions import RequestException
+from requests.exceptions import ConnectionError
+from contextlib import closing
+from bs4 import BeautifulSoup
+import json
+import os.path
+from os import getcwd
+from os import mkdir
+from pathlib import Path
+from agora_analytica import (
+    instance_path
+)
+
+
+
+
+def json_to_file (lista,name):
+
+    f = open(os.path.join(instance_path(), f"{name}.json"),"w",encoding='utf-8')
```

Comment:
instance_path() palauttaa myös pathlib -objektin, joita voi tosiaan concattia sillä `/` operaattorilla. 
```suggestion
    f = open(instance_path() / f"{name}.json","w",encoding='utf-8')
```